articles/openshift/concepts-networking.md

```diff
@@ -0,0 +1,101 @@
+---
+
+# Networking in Azure Red Hat OpenShift 4
+
+This guide covers an overview of networking in Azure Red Hat OpenShift 4 clusters, along with a diagram and a list of important endpoints.
+
+For more details on core OpenShift networking concepts, please see the [Azure Red Hat OpenShift 4 networking documentation](https://docs.openshift.com/aro/4/networking/understanding-networking.html).
+
+## Networking Concepts in Azure Red Hat OpenShift
+
+### Networking Basics in OpenShift
+
+OpenShift Software Defined Networking (SDN) is used to configure an overlay network using Open vSwitch (OVS), an OpenFlow implementation based on Container Network Interface (CNI). The SDN supports different plugins, and the plugin being used in Azure Red Hat OpenShift 4.3 is Network Policy. All network communication is managed by the SDN, so no extra routes are needed on the your Virtual Networks (VNets).
+
+### Azure Red Hat OpenShift Networking Specifics
+
+The following features are specific to Azure Red Hat OpenShift:
+* Bring your own VNet is supported.
+* Pod and Service CIDRs are configurable.
+* Nodes and Masters are in different subnets.
+* Nodes and Masters VNet subnets should be minimum /2.
+* Pod CIDR should be minimum /18 (the CIDR is not routable similar to kubenet).
+* Each node is allocated /23 subnet for its pods (you can not change this value).
+* You can not attach a pod to multiple networks (OpenShift feature).
+* You can not configure Egress static IP (OpenShift feature).
+
+### Network Modes
+
+The following network modes are available in Azure Red Hat OpenShift 4x:
+
+* **API Visibility** - Set the API visibility when running the [az aro create command](tutorial-create-cluster.md#create-the-cluster).
+    * "Public" - API Server is accessible by the public.
+    * "Private" - API Server assigned a private IP from the masters subnet, only accessible using connected networks (Peered VNetS, other subnets in ARO VNet). A private DNS Zone will be created on the customer behalf.
+* **Ingress Visibility** - Set the API visibility when running the [az aro create command](tutorial-create-cluster.md#create-the-cluster).
+    * "Public" routes will default to public SLB (this can be changed).
+    * "Private" routes will default to ILB (this can be changed).
+
+## Azure Red Hat OpenShift 4 networking diagram
+
+![Azure Red Hat OpenShift 4 networking diagram](./media/concepts-networking/aro4-networking-diagram.png)
+
+When you deploy Azure Red Hat OpenShift 4, your entire cluster is contained within a VNet. Within this VNet, your master nodes and workers nodes each live in their own subnet. Each subnet uses a public and internal load balancer.
+
+### Explanation of endpoints
+
+The following list covers important endpoints in an Azure Red Hat OpenShift cluster.
+
+* **aro-pls**
+    * This endpoint is used by Microsoft and Red Hat site reliability engineers to help manage your cluster.
```

Comment:
This is an Azure Private Link endpoint, used by Microsoft and Red Hat site reliability engineers to help manage the cluster